### PR TITLE
fix: add missing tls secret for local domain

### LINKED ISSUE #264: Fix the non-existent secret in app ingress
`Eirini-ingress` with `enabled_tls` feature creates a field `tls.secretName: {appname}-tls` in the `{appname}` ingress always. We do not create a certificate secret `{appname}-tls` for `omg` domain because it doesn't matter as traefik serves a default certificate when it cannot find the certificate secret. Though everything works, this is bad behavior. 

Acceptance Criteria:

1. Either make `tls.secretName: {appname}-tls` go away. 
OR
2. Create a certificate secret for the app even for local domain.

### PR DESCRIPTION
**Pull request by rohitsakala**
fixes #264

### CODE REVIEW

File: internal/cli/clients/client.go [hunk 1]
```diff
@@ -1087,7 +1099,136 @@ func (c *EpinioClient) createCertificate(appName, systemDomain string) error {
...
+				"certificate" : {
+					"commonName" : "%s.%s",
+					"isCA" : false,
+					"alternativeNames": [
+						"%s.%s"
+					],
+					"signerType" : "cluster"
+				}
+			},
+			"secretName" : "app1-tls-cert",
```
Comment (jimmykarily) [inline internal/cli/clients/client.go:1122]: hardcoded? :thinking:

Comment (jimmykarily) [reply internal/cli/clients/client.go:1122]: more below

Comment (rohitsakala) [reply internal/cli/clients/client.go]: Removed.

File: deployments/workloads.go [hunk 2]
```diff
@@ -239,6 +257,81 @@ func (w Workloads) createClusterRegistryCredsSecret(c *kubernetes.Cluster) error
...
+	}
+
+	dynamicClient, err := dynamic.NewForConfig(c.RestConfig)
+	if err != nil {
+		return err
+	}
+
+	err = dynamicClient.Resource(quarksSecretInstanceGVR).Namespace(WorkloadsDeploymentID).
+		Delete(context.Background(),
+			fmt.Sprintf("generate-ca-certificate"),
```
Comment (jimmykarily) [inline deployments/workloads.go:326]:
```suggestion
			"generate-ca-certificate",
```

Comment (rohitsakala) [reply deployments/workloads.go:326]: Done

**Review by jimmykarily — APPROVED** [review]

**Pull request merged**